### ISSUE TITLE
Suggestion: rename fanyi-dwim2 -> fanyi-dwim-at-point

### ISSUE THREAD

**Issue by merrickluo** [condy0919/fanyi.el#25]
It should be more understandable to name it `fanyi-dwim-at-point`, I was trying to write my own version before I found out `fanyi-dwim2` does the exact same thing. And of course thanks for the package, its great.

**Comment by condy0919** [comment]
Strongly agreed `fanyi-dwim2` is not a good name.

But [DWIM](https://en.wikipedia.org/wiki/DWIM) itself includes guessing about the word at point, naming it `-at-point` is redundant from my point of view.

A new name is welcome.

**Comment by merrickluo** [comment]
yeah, it seems dwim does include the idea of `at-point`, how about `fanyi-word-at-point` or adding a customization for `fanyi-dwim` to work like `fanyi-dwim2`? like `(setq fanyi-dwim-auto-word)`?
